python/tskit/tables.py

```diff
@@ -1721,89 +1720,76 @@ def delete_sites(self, site_ids, record_provenance=True):
 
     def delete_intervals(self, intervals, simplify=True, record_provenance=True):
         """
-        Returns a copy of this set of tables for which information in the
-        specified list of genomic intervals has been deleted.  Edges spanning
-        these intervals are truncated or deleted, and sites falling within them are
-        discarded.
-
-        Note that node IDs may change as a result of this operation,
-        as by default :meth:`.simplify` is called on the resulting tables to
-        remove redundant nodes. If you wish to keep node IDs stable between
-        this set of tables and the returned tables, specify ``simplify=True``.
-
-        See also :meth:`.keep_intervals`.
+        Delete all information from this set of tables which lies *within* the
+        specified list of genomic intervals. This is identical to the
+        :class:`TreeSequence` method of the same name but acts *in place* to alter
```

Comment:
More direct here (and elsewhere) to say "This is identical to :meth:`TreeSequence.delete_intervals`, but acts *in place*..."